x-pack/test/accessibility/apps/search_sessions.ts

```diff
@@ -50,6 +50,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('Session management filtered by applications meets a11y requirements', async () => {
+      await retry.waitFor(
+        'dashboards as a selection visible on app dropdown',
+        async () => await find.byCssSelector('[title="dashboards"]')
+      );
```

Comment:
`retry.wait` must return boolean, so you need to check for element to be displayed.

`async () = await find.existsByCssSelector('[title="dashboards"]');`